### PR TITLE
Added More Categories and Players

### PR DESCRIPTION
**Pull request by ashroyxi**
## ADDED INSTALL SCRIPT FOR LINUX

New categories (Only for TorrentGalaxy/TGX): 
- Anime
- Music
- AudioBook
- Porn

New players:
- mplayer
- smplayer

### CODE REVIEW

**Comment by ashroyxi** [comment]
@kaboussi 
say something i guess?

**Comment by kaboussi** [comment]
> @kaboussi say something i guess?

Sorry for the late reply.
it seems like you did a huge PR
i'm gonna look at it right now

File: stream_cli/api/torrentgalaxy.py [hunk 1]
```diff
@@ -31,15 +31,35 @@ def _set_category() -> str:
...
+                return (
+                    f"https://torrentgalaxy.to/torrents.php?c13=1&search={converted_name}&lang=0&nox=2"
+                    "&sort=seeders&order=desc"
+                )
+            elif category == "porn":
+                return (
+                    f"https://torrentgalaxy.to/torrents.php?c35=1&c34=1&search={converted_name}&lang=0&nox=2"
+                    "&sort=seeders&order=desc"
+                )
+
```
Comment (kaboussi) [inline stream_cli/api/torrentgalaxy.py:62]: here too

**Review by kaboussi — CHANGES_REQUESTED** [review]
This looks so good just some little details.
thanks for this beautiful PR, and sorry again for my late response

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#meeting requirements
+echo "$(tput setaf 4)Installing Requirements"
+sudo npm install webtorrent-cli -g
+git clone https://github.com/redelka00/stream-cli && cd stream-cli
+python3 -m venv venv && source venv/bin/activate
```
[the remark this answers is not in the document]
Comment (ashroyxi) [reply install.sh:7]: but you said support for windows is turned off #42 and i have no idea how windows works.

File: install.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+mainpath=$PWD
+
+echo "
+#!/bin/bash
+cd $mainpath
+python main.py
+
+" > stream-cli
+
+sudo cp stream-cli /usr/bin/
```
[the remark this answers is not in the document]
Comment (ashroyxi) [reply install.sh:28]: editing registry from cmd is not a good idea i guess

**Pull request closed without merging**